src/PAModel/PAConvert/Yaml/YamlLexer.cs

```diff
@@ -509,6 +518,21 @@ private YamlToken Unsupported(LineParser line, string message)
             return Error(line, message);
         }
 
+        private YamlToken UnsupportedSingleQuote(LineParser line, bool isComponent)
+        {
+            string[] lineSplit = line._line.ToString().ToLower().Split(new string[] { " as " }, StringSplitOptions.None);
```

Comment:
Why split on a `string[]` with one element instead of just splitting on the string itself?

Reply:
It seems like split take a char, char array or string array as arguments. Unless I am missing something.

Reply:
Oh yeah - must be a .NET Standard thing that it doesn't support the string overload. Works for me!